packages/puppeteer-core/src/common/BrowserConnector.ts

```diff
@@ -59,6 +59,12 @@ export interface BrowserConnectOptions {
    * @internal
    */
   protocol?: 'cdp' | 'webDriverBiDi';
+  /**
+   * Timeout setting for individual protocol (CDP) calls.
+   *
+   * @defaultValue 30000
+   */
+  protocolTimeout?: number;
```

Comment:
Can we just name this timeout?

Reply:
No, because we already have a `timeout` in the launch options that get united with the this interface and the `timeout` is the timeout for the browser launch operation.